### ISSUE TITLE
[BUG] Tweeten Doesn't Remember Resize of Window

### ISSUE THREAD

**Issue by spartibus** [MehediH/TweetenSite#26]
Tweeten Version: 0.1.160403-1300
OS: Windows 10 x64
Size of monitors: (1) 1920x1080 (2) 2048x1536

Issue: I use Tweeten on my second monitor. When I try to set the Tweeten window size to what I want (1000x1495), upon restarting the application, it resets to 1000x992. I assume it keeps resetting the size of the window to make sure Tweeten will fit on my primary monitor, even though it remembers to open on my secondary monitor.

**Comment by MehediH** [comment]
It is working perfectly here. I have tried with multiple monitors, and it still works.

**Comment by spartibus** [comment]
Does your secondary monitor have a bigger vertical resolution than your primary? I am guessing that is possibly the reason the bug occurs for me and not for you. It appears to be resizing so that it will fit on my primary monitor, even though it correctly remembers to open on my secondary monitor.

I've done a little more troubleshooting with it, and that seems to be what it's doing. It's not just vertical size, it will also resize horizontal if I make it bigger than my primary monitor's desktop resolution.

Example: I set the Tweeten window to 2006x1106 on my secondary monitor (it's not maximized/fullscreen, but manually sized by dragging the corner). I then close and re-open Tweeten, and Tweeten resets to 1906x992. It does this at any size bigger than 1906x992 for me, as I even tried to set it bigger by 1 pixel both horizontally and vertically (1907x993) and it still reset to 1906x992 upon closing and re-opening.

It actually does this behavior even if I move Tweeten over to my primary monitor. If I have it on my primary monitor, and set the Tweeten window's resolution to anything higher than 1906x992, even by a pixel, upon closing and re-opening, it will reset to that resolution.

It still does this behavior for me as of version 0.1.160508-2200. If I can provide you any more details, or explain something better, please let me know.

**Comment by MehediH** [comment]
Heh, happening here now as well. I will try to fix this. Thanks!

**Comment by MehediH** [comment]
Fixed with Tweeten 3.3

**Comment by spartibus** [comment]
Unfortunately this still isn't fixed for me. I tried both the 32bit and 64bit versions of 3.3.

**Comment by MehediH** [comment]
It is working completely fine here

**Comment by spartibus** [comment]
I just did a clean install by deleting AppData\Roaming\Tweeten, and it didn't fix it. The maximum vertical size that the tweeten window remembers is still 992 pixels.

In fact, the very first time I opened Tweeten, I moved it to my second monitor and made it bigger than 1906x992, closed it, and then when I re-opened it, it opened on my primary monitor with the size that tweeten originally opened to, instead of what and where I had set it.

Basically, Tweeten is still exhibiting the same behavior as my previous posts, of not remembering window sizes past 1906x992
